src/pdf_extract/pdf_extract.pro

```diff
@@ -50,3 +50,11 @@ LIBS += -L"$$PWD/../../bin"
 LIBS += -lpoppler-qt5
 INCLUDEPATH += $$PWD/../../
 LIBS += -lkicad
+
+ifneq ($(OS),Windows_NT)
+    UNAME_S = $(shell uname -s)
+    ifeq ($(UNAME_S),Darwin)
```

Comment:
qmake provides scopes for different OS targets:
http://doc.qt.io/qt-5/qmake-language.html#scopes-and-conditions

Maybe it would be a more "futureproof" approach.